core/services/job/orm.go

```diff
@@ -456,8 +456,21 @@ func (o *orm) CreateJob(jb *Job, qopts ...pg.QOpt) error {
 		jb.PipelineSpecID = pipelineSpecID
 
 		err = o.InsertJob(jb, pg.WithQueryer(tx))
+		if err != nil {
```

Comment:
@vyzaldysanchez Can we add some tests for this behaviour?

Reply:
This is still the same old behavior really, not sure why I changed the structure of things...